zhaquirks/xiaomi/aqara/roller_curtain_e1.py

```diff
@@ -0,0 +1,108 @@
+"""Aqara Roller Shade Driver E1 device."""
+
+from zigpy.profiles import zha
+from zigpy.zcl.clusters.closures import WindowCovering
+from zigpy.zcl.clusters.general import (
+    Alarms,
+    AnalogOutput,
+    Basic,
+    DeviceTemperature,
+    GreenPowerProxy,
+    Groups,
+    Identify,
+    MultistateOutput,
+    OnOff,
+    Ota,
+    Scenes,
+    Time,
+)
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.xiaomi import (
+    LUMI,
+    BasicCluster,
+    XiaomiAqaraRollerE1Cluster,
+    XiaomiCustomDevice,
+)
+
+
+class RollerE1AQ(XiaomiCustomDevice):
+    """Aqara Roller Shade Driver E1 device."""
+
+    def __init__(self, *args, **kwargs):
+        """Init."""
+        super().__init__(*args, **kwargs)
+
```

Comment:
can you add the simple descriptor comment to this. Look at other quirks to see an example. You can get this by removing the device completely from HA and then pairing it again.

Reply:
You mean add it to the signature?
Like this https://github.com/zigpy/zha-device-handlers/blob/79430799ddd887b416f5cca47361df6715add4e1/zhaquirks/xiaomi/__init__.py#L85
and this https://github.com/zigpy/zha-device-handlers/blob/dev/zhaquirks/xiaomi/aqara/vibration_aq1.py#L158

Reply:
Nope. Like this: < https://github.com/zigpy/zha-device-handlers/blob/71dd9e5167ea6f786301a66d9e9bec4b9d1f6032/zhaquirks/xiaomi/aqara/illumination.py#L28>

I have to go back and update the others

Reply:
OK thanks. Will add it later today or tomorrow.

Reply:
Done!